annotations.md

```diff
@@ -9,16 +9,16 @@ Annotations MUST be a key-value map where both the key and value MUST be strings
 While the value MUST be present, it MAY be an empty string.
 Keys MUST be unique within this map, and best practice is to namespace the keys.
 Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
-Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by other specifications and extensions.
+Keys using the `org.opencontainers.image` namespace are reserved and MUST NOT be used by other specifications and extensions.
```

Comment:
i'm not opposed, but wonder how to handle the reservation of `org.opencontainers` prefix as well?